tests/checkers/unittest_spelling.py

```diff
@@ -39,6 +40,20 @@ def _get_msg_suggestions(self, word: str, count: int = 4) -> str:
         suggestions = "' or '".join(self.checker.spelling_dict.suggest(word)[:count])
         return f"'{suggestions}'"
 
+    def test_spelling_dict_help(self) -> None:
+        assert "both the python package and the system dep" in _get_enchant_dict_help(
+            [], pyenchant_available=False
+        )
+        assert "need to install the system dep" in _get_enchant_dict_help(
+            [], pyenchant_available=True
+        )
+        if enchant is not None:
+            assert "Available dictionaries: " in _get_enchant_dict_help(
+                enchant.Broker().list_dicts(), pyenchant_available=True
+            )
+        else:
+            pytest.skip("'enchant' is not installed.")
```

Comment:
Not a big deal but could alternatively use the `skipif` decorator or something similar to line 34 in the current module.

Reply:
Nice catch, fixed in [8435e2b](https://github.com/PyCQA/pylint/pull/8338/commits/8435e2bcfe2f166eefa3f4be2b2b83390c0d60c9)